packages/ifpack2/src/Ifpack2_Container_def.hpp

```diff
@@ -317,6 +317,8 @@ void ContainerImpl<MatrixType, LocalScalarType>::DoGSBlock(
     //Use the KokkosSparse internal matrix for low-overhead values/indices access
     //But, can only do this if the matrix is accessible directly from host, since it's not a DualView
     using container_exec_space = typename ContainerImpl<MatrixType, LocalScalarType>::crs_matrix_type::execution_space;
+
+    // FENCE REVIEW: Not Tested
```

Comment:
Not really sure why this is here.

Reply:
We have UVM access of entries, rowmap, and values in the loop below - so I think this fence is probably required until that is refactored. This section does not have a unit test.